samples/msal-node-samples/ElectronReactTestApp/src/CachePlugin.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require('fs');
+
+import { CACHE_LOCATION } from './Constants';
+
+const beforeCacheAccess = async (cacheContext: any) => {
+    return new Promise<void>(async (resolve, reject) => {
+        if (fs.existsSync(CACHE_LOCATION)) {
+            fs.readFile(CACHE_LOCATION, 'utf-8', (err: any, data: any) => {
+                if (err) {
+                    reject();
+                } else {
+                    cacheContext.tokenCache.deserialize(data);
+                    resolve();
+                }
+            });
+        } else {
+            fs.writeFile(CACHE_LOCATION, cacheContext.tokenCache.serialize(), (err: any) => {
```

Comment:
if cache.json does not exist on disk beforehand, the sign-in flow triggers this code, and the cache.json does get created, however only the cache schema is written on it, the in-memory cache snapshot is not written. This is causing the app to hang on the sign-in screen.

It would be great if you could delete your existing cache.json and give a debug at this (make sure to enable trace logs). I assume this also pertains to ElectronTestApp. @hectormmg @sameerag any ideas?